test/unittests/libethereum/ClientTest.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/** @file ClientTest.cpp
+ * @date 2015
+ */
+
+#include <libethereum/ChainParams.h>
+#include <libethereum/ClientTest.h>
+#include <libp2p/Network.h>
+#include <libwebthree/WebThree.h>
+#include <test/tools/libtesteth/TestOutputHelper.h>
+#include <boost/test/unit_test.hpp>
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+using namespace dev::test;
+using namespace dev::p2p;
+namespace fs = boost::filesystem;
+
+std::unique_ptr<dev::WebThreeDirect> g_web3;
```

Comment:
Why do you make it global and not the member of the fixture class?

Reply:
how to access fixture from the test case?

Reply:
test cases are methods of the fixture.